Source/objects.cpp

```diff
@@ -5142,54 +5142,54 @@ void SyncBreakObj(int pnum, int oi)
 		BreakBarrel(pnum, oi, 0, true, false);
 }
 
-void SyncCrux(int i)
+void SyncCrux(const ObjectStruct &crux)

```

Comment:
I know it is unrelated to the change, but I really dislike seeing this kind of thing. An operation that is specific to a certain type of object, but then passing an `ObjectStruct` which could actually represent any type of object (and not just the one where the operation would make sense).

This is where actual OO and polymorphism seems to be the way to go. Either that, or the method should be more general and work on any object.